source/_integrations/openuv.markdown

```diff
@@ -31,31 +31,59 @@ trained medical professional.
-component does not automatically query the API for new data after it initially
-loads. To request new data, the `update_data` service may be used.
-</div>
-
-<div class='note warning'>
-Each use of the `update_data` service will consume 2 API calls from the daily quota
-(since it performs the same tasks as back-to-back calls of the `update_uv_index_data` and
-the `update_protection_data` services).
-</div>
-
 {% include integrations/config_flow.md %}
 
 ## Sensors
 
+<div class='note info'>
+The approximate number of minutes of a particular skin type can be exposed to
+the sun before burning/tanning starts is based on the
+[Fitzpatrick scale](https://en.wikipedia.org/wiki/Fitzpatrick_scale).
+</div>
+
 | Name | Type | Value |
 |------|------|-------|
 | Current Ozone Level | Sensor | ozone level in du (Dobson Units) |
 | Current UV Index | Sensor | UV Index (numerical value) |
 | Current UV Level | Sensor | UV Level (as literal) |
 | Max UV Index | Sensor | max UV Index for the day (at solar noon) |
 | Protection Window | Binary Sensor | whether sunblock protection should be used |
+| Skin Type 1 Safe Exposure Time | Sensor | the amount of time Fitzpatrick skin type 1 can be in the sun unprotected |
+| Skin Type 2 Safe Exposure Time | Sensor | the amount of time Fitzpatrick skin type 2 can be in the sun unprotected |
+| Skin Type 3 Safe Exposure Time | Sensor | the amount of time Fitzpatrick skin type 3 can be in the sun unprotected |
+| Skin Type 4 Safe Exposure Time | Sensor | the amount of time Fitzpatrick skin type 4 can be in the sun unprotected |
+| Skin Type 5 Safe Exposure Time | Sensor | the amount of time Fitzpatrick skin type 5 can be in the sun unprotected |
+| Skin Type 6 Safe Exposure Time | Sensor | the amount of time Fitzpatrick skin type 6 can be in the sun unprotected |
+
+## Updating Data
+
+<div class='note warning'>
+OpenUV does _not_ automatically update data for its entities! Users must manually
+update data via the `homeassistant.update_entity` service.
+</div>
+
+Beginning February 1, 2019, the "Limited" plan (which is what new users are given by
+default) is limited to 50 API requests per day. Because different API plans and
+locations will have different requirements, the `openuv` component does not automatically
+query the API for new data after it initially loads. To request new data, the
+`homeassistant.update_entity` service may be used.
+
+Note that in the case of UV and ozone data, selecting any one of:
+
+* Current Ozone Level
+* Current UV Index
+* Current UV Level
+* Max UV Index
+* Skin Type 1 Safe Exposure Time
+* Skin Type 2 Safe Exposure Time
+* Skin Type 3 Safe Exposure Time
+* Skin Type 4 Safe Exposure Time
+* Skin Type 5 Safe Exposure Time
+* Skin Type 6 Safe Exposure Time
+
+...as the target for the `homeassistant.update_entity` service will update the data for
+_all_ of these entities. This is important to keep in mind, as updating each entity
+separately will result in wasted API calls.
```

Comment:
Can you add a throttle on update that a) aborts update when it's in progress and b) does not allow updating more than once every 15 minutes?